tests/run.bats

```diff
@@ -19,6 +19,7 @@ load helpers
 	buildah run        $cid cp /tmp/randomfile /tmp/other-randomfile
 	test -s $root/tmp/other-randomfile
 	cmp ${TESTDIR}/randomfile $root/tmp/other-randomfile
+	buildah run $cid -- rm -f /tmp/randomfile /tmp/other-randomfile
```

Comment:
Might be better to test with 'echo' here, so that we can capture the output and verify that it matches what we expect (and specifically, that it doesn't include "--" when it shouldn't).  Otherwise LGTM.